nchs_mortality/tests/test_pull.py

```diff
@@ -8,70 +8,91 @@
-                "covid_deaths": [2],
-                "pneumonia_and_covid_deaths": [4],
-                "pneumonia_influenza_or_covid_19_deaths": [8]
-            }))
-        expected = pd.DataFrame({
-            "timestamp": [1],
-            "covid_19_deaths": [2],
-            "pneumonia_and_covid_19_deaths": [4],
-            "pneumonia_influenza_or_covid_19_deaths": [8]
-        })
+            pd.DataFrame(
+                {
+                    "start_week": [1],
+                    "covid_deaths": [2],
+                    "pneumonia_and_covid_deaths": [4],
+                    "pneumonia_influenza_or_covid_19_deaths": [8],
+                }
+            )
+        )
+        expected = pd.DataFrame(
+            {
+                "timestamp": [1],
+                "covid_19_deaths": [2],
+                "pneumonia_and_covid_19_deaths": [4],
+                "pneumonia_influenza_or_covid_19_deaths": [8],
+            }
+        )
         pd.testing.assert_frame_equal(expected, df)
 
     def test_good_file(self):
-        df = pull_nchs_mortality_data(TOKEN, "test_data.csv")
+        df = pull_nchs_mortality_data(SOCRATA_TOKEN, "test_data.csv")
 
         # Test columns
-        assert (df.columns.values == [
-                'covid_19_deaths', 'total_deaths', 'percent_of_expected_deaths',
-                'pneumonia_deaths', 'pneumonia_and_covid_19_deaths',
-                'influenza_deaths', 'pneumonia_influenza_or_covid_19_deaths',
-                "timestamp", "geo_id", "population"]).all()
+        assert (
+            df.columns.values
+            == [
+                "covid_19_deaths",
+                "total_deaths",
+                "percent_of_expected_deaths",
+                "pneumonia_deaths",
+                "pneumonia_and_covid_19_deaths",
+                "influenza_deaths",
+                "pneumonia_influenza_or_covid_19_deaths",
+                "timestamp",
+                "geo_id",
+                "population",
+            ]
+        ).all()
 
         # Test aggregation for NYC and NY
         raw_df = pd.read_csv("./test_data/test_data.csv", parse_dates=["start_week"])
         raw_df = standardize_columns(raw_df)
         for metric in METRICS:
-            ny_list = raw_df.loc[(raw_df["state"] == "New York")
-                                & (raw_df[metric].isnull()), "timestamp"].values
-            nyc_list = raw_df.loc[(raw_df["state"] == "New York City")
-                                & (raw_df[metric].isnull()), "timestamp"].values
-            final_list = df.loc[(df["geo_id"] == "ny")
-                                & (df[metric].isnull()), "timestamp"].values
+            ny_list = raw_df.loc[
+                (raw_df["state"] == "New York") & (raw_df[metric].isnull()), "timestamp"
+            ].values
+            nyc_list = raw_df.loc[
+                (raw_df["state"] == "New York City") & (raw_df[metric].isnull()),
+                "timestamp",
```

Comment:
Should trailing comma at end of list be consistent (i.e. either always end in a comma like line 65 or end with no comma like line 61)?
```suggestion
                "timestamp"
```

Reply:
oh whoops. My styler was unhappy with this file, and I accidentally included it's changes.